README.md

```diff
@@ -138,15 +138,48 @@ The expected payload structure is as follows:
 
 ## Streamlit Application
 
-Once the deployment has completed, run the following command to start a streamlit application.
+To deploy streamlit application on AKS, we first build a Docker image for streamlit application and push the image to ACR.
 
-> Note: you need to first obtain the model ingress IP as described above.
+> Note: Run the following command from the root of the project.
+
+Verify that you are in the root of the project.
+
+```bash
+pwd
+
+/home/username/MindGPT
+```
+
+We build and push the streamlit application to ACR. This image will be used by Kubernetes deployment.
```

Comment:
Before that, we need to set two environment variables. One for the ACR registry URI and another for the registry name. We'll use Matcha to do this: